### PR TITLE
feat: add UI french translations

### PR DESCRIPTION
**Pull request by jufab**
Add french translations in UI.

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
Thanks for offering this @jufab! We've run into problems with translations in the past due to lack of participation after first merge. Can you help find 2 other french-speaking folks to review this and possibly help maintain this? cc @fedj

**Comment by codefromthecrypt** [comment]
just sent a tweet on our account to help rally support also

**Comment by jufab** [comment]
No problem @adriancole 
I used your post on Tweeter to find other french people 👍

File: zipkin-lens/src/translations/fr/messages.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "Dependencies": "Dépendances",
+  "Depth": "Profondeur",
+  "Download JSON": "Télécharger JSON",
+  "Duration": "Durée",
+  "End Time": "Fin",
+  "Find a trace": "Trouver une trace",
+  "Limit": "Limite",
+  "None": "Aucun",
+  "Parent ID": "ID Parent",
+  "Please select criteria in the search bar. Then, click the search button.": "S'il vous plaît sélectionner un critère dans la barre de recherche. Et, cliquer sur le bouton de recherche",
```
Comment (ImFlog) [inline zipkin-lens/src/translations/fr/messages.json]: I would translate the `Then` by `Puis` and also I think the verbs should be conjuged (sélectonnez VS sélectionner) ?
```suggestion
  "Please select criteria in the search bar. Then, click the search button.": "S'il vous plaît sélectionnez un critère dans la barre de recherche. Puis cliquez sur le bouton de recherche.",
```

File: zipkin-lens/src/translations/fr/messages.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "Please select the start and end time. Then, click the search button.": "S'il vous plaît sélectionner la date de début et de fin. Et, cliquer sur le bouton de recherche",
```
Comment (ImFlog) [inline zipkin-lens/src/translations/fr/messages.json]: Same remark as above.

File: zipkin-lens/src/translations/fr/messages.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  "Relative Time": "Temps Relatif",
+  "Root": "Racine",
+  "Search Dependencies": "Rechercher des dépendances",
+  "Search Traces": "Rechercher des traces",
+  "Searching has been disabled via the searchEnabled property. You can still view specific traces of which you know the trace id by entering it in the \"Trace ID...\" textbox on the top-right.": "La recherche a été désactivée via la propriété searchEnabled. Vous pouvez toujours afficher des traces spécifiques dont vous connaissez l'identifiant de trace en le saisissant dans la zone de texte \"ID Trace ... \" en haut à droite.",
+  "Services": "Services",
+  "Span ID": "ID Section",
```
Comment (ImFlog) [inline zipkin-lens/src/translations/fr/messages.json:23]: Should the span word be translated ? I think section (I've actually never find a good translation).

File: zipkin-lens/src/translations/fr/messages.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  "Start Time": "Début",
+  "Support": "Support",
+  "Tags": "Étiquettes",
+  "Total Spans": "Sections Total",
+  "Trace ID": "ID Trace",
+  "Upload JSON": "Charger un JSON",
+  "View Logs": "Voir les journaux",
+  "Zipkin": "Zipkin",
+  "hide annotations": "Masquer les annotations",
+  "show all annotations": "Afficher toutes les annotations",
```
Comment (ImFlog) [inline zipkin-lens/src/translations/fr/messages.json]: I think the capital letters are not needed here ? Not done for the other languages.

File: zipkin-lens/src/translations/fr/messages.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  "End Time": "Fin",
+  "Find a trace": "Trouver une trace",
+  "Limit": "Limite",
+  "None": "Aucun",
+  "Parent ID": "ID Parent",
+  "Please select criteria in the search bar. Then, click the search button.": "S'il vous plaît sélectionner un critère dans la barre de recherche. Et, cliquer sur le bouton de recherche",
+  "Please select the start and end time. Then, click the search button.": "S'il vous plaît sélectionner la date de début et de fin. Et, cliquer sur le bouton de recherche",
+  "Relative Time": "Temps Relatif",
+  "Root": "Racine",
+  "Search Dependencies": "Rechercher des dépendances",
```
Comment (ImFlog) [inline zipkin-lens/src/translations/fr/messages.json]: Given the UI I would have said "Rechercher les dépendances" ?

**Review by ImFlog — COMMENTED** [review]
Just a few remarks (happy to discuss).

Also first time I see the new UI. Great job @tacigar, that's awesome !

File: zipkin-lens/src/translations/fr/messages.json (hunk 1 above)
Comment (jufab) [reply zipkin-lens/src/translations/fr/messages.json]: Agree with you for the verbs, and replace "Et" by "Puis" 👍

File: zipkin-lens/src/translations/fr/messages.json (hunk 2 above)
Comment (jufab) [reply zipkin-lens/src/translations/fr/messages.json]: Same too :)

File: zipkin-lens/src/translations/fr/messages.json (hunk 3 above)
Comment (jufab) [reply zipkin-lens/src/translations/fr/messages.json:23]: I agree with you, I would like to conserve span too but I want to translate all terms... 
I saw Spanish translation who use "section" and I use the same idea.

File: zipkin-lens/src/translations/fr/messages.json (hunk 4 above)
Comment (jufab) [reply zipkin-lens/src/translations/fr/messages.json]: Yes, you right!! 👍

File: zipkin-lens/src/translations/fr/messages.json (hunk 5 above)
Comment (jufab) [reply zipkin-lens/src/translations/fr/messages.json]: At first, I translated by " Rechercher les dépendances" but after use UI, I replaced by "Rechercher des dépendances"... 
I have doubt about use "les" ou "des" but I'm going to follow your idea

**Comment by jufab** [comment]
@ImFlog Thanks for your review, I pushed new version.
Just I have no better word than "section"... :D

**Review by ImFlog — APPROVED** [review]

**Comment by codefromthecrypt** [comment]
I see three independent folks interested in this besides me, as well two who can help maintain this. Let's do it!

**Pull request merged**